### PR TITLE
Support Paligemma google/paligemma-3b-mix-224

### PR DESCRIPTION
**Pull request by vincentzed**
## Motivation

This is step 0 for support pi0-fast autoregressive VLA sprint.

## Modifications
Support google/paligemma-3b-mix-224

`python -m sglang.launch_server --model google/paligemma-3b-mix-224 --skip-server-warmup --attention-backend torch_native`

It has no chat template, so only use generate API. And skip warmup because warmup happen and try to use chat template - causes crash. And we only support prefix lm bidirectional decode for torch_native, also, it is only used in MM case. 

Validated with the 
`url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/car.jpg?download=true"`

`Un auto azul estacionado frente a un edificio.`

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review Process

1. Ping Merge Oncalls to start the PR flow. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - `/tag-run-ci-label`, `/rerun-failed-ci`, `/tag-and-rerun-ci`
4. After green CI and required approvals, ask Merge Oncalls to merge.

### CODE REVIEW

File: python/sglang/srt/models/paligemma.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                vision_output = self.vision_tower(pixel_values=pixel_value)
+                batch_vision_outputs.append(vision_output)
+
+            if batch_vision_outputs:
+                vision_outputs_cat = torch.cat(batch_vision_outputs, dim=0)
+                projected_features = self.multi_modal_projector(vision_outputs_cat)
+                # Scale by hidden_size^(-0.5) to compensate for the sqrt(hidden_size)
+                # scaling applied later in GemmaModel.forward
+                projected_features = projected_features * (self.config.hidden_size**-0.5)
+                final_features_list.append(projected_features)
```
[the remark this answers is not in the document]
Comment (vincentzed) [reply python/sglang/srt/models/paligemma.py:172]: Scaling
# Scale by hidden_size^(-0.5) to compensate for the sqrt(hidden_size)
                # scaling applied later in GemmaModel.forward

This is pi0-fast specific, we can remove it, for normal inference.
